src/main/java/hudson/plugins/sonar/SonarBuildWrapper.java

```diff
@@ -112,7 +111,7 @@ public SonarInstallation getInstallation() {
 
   @Override
   public Environment setUp(AbstractBuild build, Launcher launcher, BuildListener listener) throws IOException, InterruptedException {
-    if (!SonarRunnerBuilder.isSonarInstallationValid(getInstallationName(), listener)) {
+    if (!SonarInstallation.isValid(getInstallationName(), listener)) {
       return new Environment() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this a named "static" inner class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.internal.sonarsource.com/coding_rules#rule_key=squid%3AS2694)